tests/common_vision_utils.py

```diff
@@ -99,6 +102,33 @@ def load_images(data):
     return np.array(images)
 
 
+def get_base64_string_from_path(img_path):
+    """Load and convert pillow image to base64-encoded image
+
+    :param img_path: image path
+    :type img_path: str
+    :return: base64-encoded image
+    :rtype: str
+    """
+    img = Image.open(img_path)
+    imgio = BytesIO()
+    img.save(imgio, img.format)
+    img_str = base64.b64encode(imgio.getvalue())
+    return img_str.decode("utf-8")
+
+
+def load_images_for_automl_images(data):
```

Comment:
maybe call this "load_base64_images"? we might want to use this for other models in the future